lib/player.js

```diff
@@ -2554,6 +2554,20 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
       movePlayhead: (delta) => { mediaElement.currentTime += delta; },
     });
 
+    if (this.cmcdManager_) {
```

Comment:
I'm thinking if it wouldn't be better if cmcdManager had the video element and created all the listeners it needs (thinking that more may be needed in the future). @littlespex what do you think?

Reply:
I agree. The `cmcdManager` should be as self-contained as possible.